frontend/src/audio.html

```diff
@@ -112,7 +128,7 @@
                 <span class="hidden lg:inline">Download</span>
               </a>
             </span>
-            <span class="p-1">
+            <span class="pr-1 pl-1">
```

Comment:
Rather than adding left and right padding to the middle element, I suggest you to add spacing class to the parent element. See how it's done in the tag line.